bentoml/_internal/frameworks/pytorch.py

```diff
@@ -170,10 +184,12 @@ def forward(self, inputs):
     )
     weight_file = _model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
     if isinstance(model, torch.jit.ScriptModule):  # type: ignore[reportPrivateUsage]
+        _model.info.options["_model_format"] = "torchscript:v1"
```

Comment:
should we save this information in `ModelInfo.options` or `ModelInfo.context`?

Reply:
`options` seems find for me

Reply:
chatted with Chaoyu, lets put it under context

Reply:
done